### ISSUE TITLE
Variable products with more than one attribute are displayed without the attributes

### ISSUE THREAD

**Issue by grigorijurasov** [woocommerce/woocommerce-ios#1127]
**Describe the bug**
Variable products made up of more than one attribute are displayed without the attributes

**To Reproduce**
1. Create a variable product with one attribute, create variations from this attribute
2. Purchase it
3. The selected attribute term shows up in the name in the app
4. Add a second attribute, re-create variations
5. Purchase the product (two variation dropdowns should appear on the product page)
6. Only the name of a parent variable product but not the variation itself shows up in the app.

**Screenshots**

1 attribute:
<img width="280" alt="Screenshot 2019-07-26 at 13 37 31" src="https://user-images.githubusercontent.com/13084901/61946195-988a1c80-afaa-11e9-9f69-5d7733aff1f3.png">

2 attributes:

<img width="288" alt="Screenshot 2019-07-26 at 13 38 25" src="https://user-images.githubusercontent.com/13084901/61946252-b6578180-afaa-11e9-96a0-8f32017d2fa6.png">

**Expected behavior**

When a variable product is purchased - I'd expect to see which particular variation is it

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone SE
- iOS version: 
- WooCommerce iOS version: 2.2.2.0

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>
https://github.com/woocommerce/woocommerce-android/issues/1286

**Comment by mindgraffiti** [comment]
Per @nbradbury's findings, this is a bug in the WC v3 API. Ref. https://github.com/woocommerce/woocommerce-rest-api/issues/9

**Comment by rachelmcr** [comment]
This was raised again by a beta tester, with the emphasis that this makes it impossible to fulfill orders using the app:

> No variable information so order cannot be fulfilled from the app.

**Comment by zwiesman31** [comment]
Reported in [2538895-zen](https://woothemes.zendesk.com/agent/tickets/2538895) as well.

**Comment by rezzap** [comment]
This issue comes up often in the app support tickets. We thought it was expected behaviour but it seems like this is a bug. 

Example ticket: 3226544-zen

**Comment by shiki** [comment]
This is in the works. We're waiting for this REST API change to be accepted and merged: https://github.com/woocommerce/woocommerce-rest-api/pull/229

**Comment by joweber123** [comment]
Mentioned here as well - 3234437-zen

**Comment by reginabally** [comment]
Reported in 3401252-zen.

**Comment by shiki** [comment]
The necessary REST API changes (https://github.com/woocommerce/woocommerce-rest-api/pull/229) have been merged into Core and will be available with WC 4.7.0.
